### PR TITLE
Nwn.python

### PR DESCRIPTION
**Pull request by milliorn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,19 @@ docker-compose up
 ```
 
 and nwserver will spin up.
+
+## Install Builder
+
+You can ignore the `How to Launch` section up to `docker-compose up` by using `Builder`. To install is simply
```
Comment (zunath) [inline README.md]: Some grammar issues here: "To install is simply adding this folder to your solution directory." -> "To install it simply add this folder to your solution directory."

File: README.md [hunk 2]
```diff
@@ -30,3 +30,19 @@ docker-compose up
...
+## Install Builder
+
+You can ignore the `How to Launch` section up to `docker-compose up` by using `Builder`. To install is simply 
+adding this folder to your solution directory. From there you will need to add this snippet to your `.csproj` file.
+```
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+    <Exec Command="$(SolutionDir)Builder\deployBuild.exe -b $(ProjectDir)$(OutDir)" />
+  </Target>
+``` 
+After that runs your files will be added to `dotnet` folder. From there you `cd` to the directory that has `docker-compose.yml``
```
Comment (zunath) [inline README.md]: It's probably more accurate to say "Any time your project builds, the binary files will be automatically copied to your dotnet folder" or something to that effect.

File: README.md [hunk 3]
```diff
@@ -30,3 +30,19 @@ docker-compose up
 ```
 
 and nwserver will spin up.
+
+## Install Builder
+
+You can ignore the `How to Launch` section up to `docker-compose up` by using `Builder`. To install is simply 
+adding this folder to your solution directory. From there you will need to add this snippet to your `.csproj` file.
+```
+  <Target Name="PostBuild" AfterTargets="PostBuildEvent">
```
Comment (zunath) [inline README.md:39]: Do we want to add this to the actual template csproj too, or leave it as an option?

Comment (milliorn) [reply README.md:39]: Originally I had this in the csproj in a commit. When I edited the readme I realize maybe someone doesn't want that in there project file so I took it out and made note of it in the readme instead.

**Pull request merged**